docs(vitals) add link to InfluxDB topic (#2414)
Kong/docs.konghq.com: app/enterprise/2.1.x/vitals/vitals-reports.md

## Prerequisites

>    * InfluxDB database. See the [Vitals with InfluxDB](https://docs.konghq.com/enterprise/{{page.kong_version}}/vitals/vitals-influx-strategy/) topic for more information.  

Reviewer: ```suggestion
   * InfluxDB database installed and configured. For more information, see [Vitals with InfluxDB](https://docs.konghq.com/enterprise/{{page.kong_version}}/vitals/vitals-influx-strategy/).  
```
